src/content/learn/index.md

```diff
@@ -456,7 +458,7 @@ function MyButton() {
 
 ```
 
-Then, *pass the state down* from `MyApp` to each `MyButton`, together with the shared click handler. You can pass information to `MyButton` using the JSX curly braces, just like you previously did with built-in tags like `<img>`:
+Kemudian, *oper state ke bawah* dari `MyApp` ke setiap `MyButton`, bersama dengan penangan klik bersama (*shared click handler*). Anda dapat mengoper informasi ke `MyButton` menggunakan kurung kurawal JSX, seperti yang sebelumnya Anda lakukan dengan *tag* bawaan seperti `<img>`:
```

Comment:
@resir014 @zainfathoni @mazipan @r17x 

This is quite tricky. Was there any merged PR with this phrase?

Which one to use:

- `*shared click handler*`
- `penangan (*handler*) klik bersama`
- `penangan klik bersama (*shared click handler*)`
- ?